component/web/oauth-common/src/main/java/org/gatein/security/oauth/openid/OpenIdProcessorImpl.java

```diff
@@ -73,6 +77,10 @@ public OpenIdProcessorImpl(ExoContainerContext context, InitParams params, Secur
         } else {
             applicationName = "GateIn portal";
         }
+        ValueParam localOidcServer = params.getValueParam("oidcServer");
+        if (localOidcServer != null) {
+            this.oidcServer = localOidcServer.getValue();
+        }
```

Comment:
Not sure if we need it for this task. 
Perhaps when we make the third task about validating the jwt, but not needed in this current task